contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -166,6 +167,26 @@ func (h *htlcIncomingContestResolver) Resolve() (ContractResolver, error) {
 	}
 }
 
+func (h *htlcIncomingContestResolver) report() *ContractReport {
+	// No locking needed as these values are read-only.
```

Comment:
Can likely elevate this comment to a "NOTE: ", above for the `godoc` comment for the method itself.

Reply:
I like to keep it inside the function, as it is an implementation detail.